### DISCUSSION TITLE
TP1 Macowins. Error con los packages

### DISCUSSION THREAD

**Discussion by tobiascastaneda2000** [dds-jv/foro#350] [Consultas de herramientas] [answered]
Buenas Tardes! 

Me baje el repositorio del tp macowins desde este link https://github.com/dds-utn/2022-tpi-macowins-tobiascastaneda2000
El problema es que cuando me lo bajo y lo abro en el Intellij, no lo hace como proyecto Maven, por lo que a la hora de querer abrir el package simplemente no lo tiene (Notese que cuando abris un proyecto Maven, la carpeta Java es de color azul, mientras que al bajar el repositorio de Macowins y abrirlo, me sale de un color blanco normal)

Puedo crear un proyecto Maven desde cero  y trabajar en el, pero el problema es que te utilizar el proyecto del Macowins que me baje para poder hacer los commits y subirlos al repositorio.

Por lo que quería preguntar como podría cambiar mi proyecto ya creado a un proyecto Maven o en todo caso como podría resolver este inconveniente

Disculpe las molestias. Espero su respuesta, Saludos!!

![Proyecto Macowins que no es Maven](https://user-images.githubusercontent.com/62578744/164579871-eada6039-6e6c-430e-932c-0725fcb6d6f1.jpg)
Captura del tp Macowins, por alguna razón no viene como proyecto Maven (Ya borre y volvi a bajar el repositorio y el resultado es el mismo) por lo que no me permite crear packages para empezar a codear.

![Proyecto Maven cualquiera](https://user-images.githubusercontent.com/62578744/164579986-b27bb3de-a0c4-435d-bc1d-26b085b2833f.jpg)
Un proyecto Maven cualquiera, cuando creas un proyecto nuevo te da la opción que sea en Maven

**Comment by gastonprieto** [comment, marked as answer]
Buenas @tobiascastaneda2000 

Creo que puede ser porque cuando importaste la carpeta no elegiste en la que esta el pom.xml sino una superior, fijate que en la primer foto tenes 1 nivel extra antes de llegar al pom.

**Reply by gastonprieto** [reply]
te sirvio @tobiascastaneda2000?

**Reply by gastonprieto** [reply]
lo doy por cerrado, cualquier cosa volvenos a preguntar @tobiascastaneda2000
